packages/components/inputs/search-text-input/src/search-text-input.story.js

```diff
@@ -16,6 +17,11 @@ storiesOf('Components|Inputs', module)
     },
   })
   .add('SearchTextInput', () => {
+    const ref = useRef(null);
+
+    const handleReset = useCallback(() => {
+      ref.current?.focus();
+    }, []);
```

Comment:
I added this to the story, but perhaps some users may find it misleading? 🤔 WDYT

Reply:
Personally I wouldn't change anything in the Story because of the `ref` change, but it's up to you.

Reply:
Yes, that makes sense. Reverted here 70558d239e9db12d0349664b39d6703303aa5efe